x-pack/plugins/security_solution/cypress/tasks/api_calls/timelines.ts

```diff
@@ -5,7 +5,88 @@
  * 2.0.
  */
 
-import { CompleteTimeline } from '../../objects/timeline';
+import { CompleteTimeline, IndicatorMatchtimelineTemplate } from '../../objects/timeline';
+
+export const createIndicatorMatchTimelineTemplate = (timeline: IndicatorMatchtimelineTemplate) =>
+  cy.request({
+    method: 'POST',
+    url: 'api/timeline',
+    body: {
+      timeline: {
+        columns: [
+          { columnHeaderType: 'not-filtered', id: 'threat.indicator.matched.type' },
+          { columnHeaderType: 'not-filtered', id: 'threat.indicator.matched.field' },
+          { columnHeaderType: 'not-filtered', id: 'threat.indicator.matched.atomic' },
+        ],
+        dataProviders: [
```

Comment:
Would it be possible to leverage the [prebuilt CTI template](https://github.com/elastic/kibana/pull/95840) as opposed to manually creating one, here? There looks to be a lot of overlap, and testing the prebuilt template seems to me like a more useful/comprehensive test.

Reply:
Same as the other comment, for me was easier/faster to do it like this, but I'll investigate to improve it in another iteration.